website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -56,7 +56,17 @@ models:
     tests: [] # todo! add tests later
     config: ...
 ```
-Some options that could previously be specified before a sub-command can now only be specified afterward. For example, `dbt --profiles-dir . run` isn't valid anymore, and instead, you need to use `dbt run --profiles-dir .`
+
+Some options that could previously be specified _after_ a sub-command can now only be specified _before_. This includes the inverse of the command, `--write-json` and `--no-write-json`, for example.  The list of affected commands are:
+
+| --cache-selected-only | --debug | --deprecated-print | --enable-legacy-logger| --fail-fast |
+|--------------|--------------|-----------------|-----------|---------------------------|
+| --log-cache-events  | --log-format | --log-level | --log-path | --macro-debugging |
+| --partial-parse      | --populate-cache | --print      | --printer-width | --quiet |
+| --record-timing-info | --send-anonymous-usage-stats | --single-threaded | --static-parser | --use-colors |
+| --use-experimental-parser  | --version | --version-check | --warn-error | --write-json | 
```

Comment:
it might just be me but i'm not sure how to interpret this table, like what's the relationship between the header row and the commands underneath it. 

screenshot: 

![Screenshot 2023-07-31 at 3 21 41 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/d7490d9b-8672-4248-be72-e45986fb06ce)

Reply:
Sorry, brain was on autopilot from the permissions table. This should render better. It won't have any headers, but we're using the table to display them in a slightly more condensed way than a list. But, if this doesn't look appealing, we can just use the aforementioned list format.

Reply:
I don't have a strong opinion on how to present this. It's a lot of items, so will be an eyeful no matter what.

If you want, you can always hide it within a toggle as well, like this:

<details>
<summary>Table-formatted listing</summary>


|              |              |                 |           |                           |
|--------------|--------------|-----------------|-----------|---------------------------|
| --cache-selected-only | --debug | --deprecated-print | --enable-legacy-logger| --fail-fast |
| --log-cache-events  | --log-format | --log-level | --log-path | --macro-debugging |
| --partial-parse      | --populate-cache | --print      | --printer-width | --quiet |
| --record-timing-info | --send-anonymous-usage-stats | --single-threaded | --static-parser | --use-colors |
| --use-experimental-parser  | --version | --version-check | --warn-error | --write-json | 
</details>

or this:

<details>
<summary>Code-formatted listing</summary>

```
--write-json / --no-write-json
--warn-error-options
--warn-error
--version-check / --no-version-check
--version / -V / -v
--use-experimental-parser / --no-use-experimental-parser
--use-colors-file / --no-use-colors-file
--use-colors / --no-use-colors
--static-parser / --no-static-parser
--single-threaded / --no-single-threaded
--send-anonymous-usage-stats / --no-send-anonymous-usage-stats
--record-timing-info / -r
--quiet / -q / --no-quiet
--printer-width
--print / --no-print
--populate-cache / --no-populate-cache
--partial-parse-file-path
--partial-parse / --no-partial-parse
--macro-debugging / --no-macro-debugging
--log-path
--log-level-file
--log-level
--log-format-file
--log-format
--log-cache-events / --no-log-cache-events
--fail-fast / -x / --no-fail-fast
--enable-legacy-logger / --no-enable-legacy-logger
--deprecated-print / --deprecated-no-print
--debug / -d / --no-debug
--cache-selected-only / --no-cache-selected-only
```
</details>

or this:

<details>
<summary>Bullet item listing</summary>

- `--write-json / --no-write-json`
- `--warn-error-options`
- `--warn-error`
- `--version-check / --no-version-check`
- `--version / -V / -v`
- `--use-experimental-parser / --no-use-experimental-parser`
- `--use-colors-file / --no-use-colors-file`
- `--use-colors / --no-use-colors`
- `--static-parser / --no-static-parser`
- `--single-threaded / --no-single-threaded`
- `--send-anonymous-usage-stats / --no-send-anonymous-usage-stats`
- `--record-timing-info / -r`
- `--quiet / -q / --no-quiet`
- `--printer-width`
- `--print / --no-print`
- `--populate-cache / --no-populate-cache`
- `--partial-parse-file-path`
- `--partial-parse / --no-partial-parse`
- `--macro-debugging / --no-macro-debugging`
- `--log-path`
- `--log-level-file`
- `--log-level`
- `--log-format-file`
- `--log-format`
- `--log-cache-events / --no-log-cache-events`
- `--fail-fast / -x / --no-fail-fast`
- `--enable-legacy-logger / --no-enable-legacy-logger`
- `--deprecated-print / --deprecated-no-print`
- `--debug / -d / --no-debug`
- `--cache-selected-only / --no-cache-selected-only`

</details>

Reply:
thanks so much! i like the "code-formatted list" idea using `details` (collapsible).

@dbeatty10 / @matthewshaver : instead of using `/` , i used `|` to indicate "or" and `,` to indicate "and" following Linux man page conventions. is this ok for dbt users?

Reply:
Good idea!

Yes, that should work well 🤓